android/build.gradle

```diff
@@ -57,6 +57,8 @@ dependencies {
   // TODO remove this library when default LinearLayoutManager will be fixed.
   compile 'org.solovyev.android.views:linear-layout-manager:0.5@aar'
   compile 'com.timehop.stickyheadersrecyclerview:library:0.4.3@aar'
+  //Glide
+  compile 'com.github.bumptech.glide:glide:3.7.0'
```

Comment:
@ygorshenin Ideally, we shouldn't use any 3party lib, but in this case I think it's okay, because I know how much code should be written to implement all gallery stuff by our forces. I'm concerned about these points:

1. As @ygorshenin already mentioned, what about licence compliance !? I'll check it with @burivuh 
2. What about size of apk after embedding this library into our application? I know this library supports playing giff anymation, but we don't need it.

If licence compliance and apk size change satisfy us - I'm ok with integrating the Glide library. If not - we should implement image loading ()https://developer.android.com/training/displaying-bitmaps/display-bitmap.html ) by our forces.